presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestSortFieldUtils.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.iceberg.NullOrder;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.SortOrder;
+import org.apache.iceberg.types.Types;
+import org.intellij.lang.annotations.Language;
+import org.testng.annotations.Test;
+
+import java.util.function.Consumer;
+
+import static com.facebook.presto.iceberg.SortFieldUtils.parseSortFields;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.testng.Assert.assertEquals;
+
+public class TestSortFieldUtils
+{
+    @Test
+    public void testParse()
```

Comment:
This is about a hundred different tests, not a unit test. Split these into individual tests that pass or fail independently, or perhaps use a parameterized test.

Reply:
Done